apps/web/src/components/featured-pack/featured-pack.tsx

```diff
@@ -73,7 +73,7 @@ export default function HomeTemplate({
         {/* Image */}
         <div className={css.featuredImage}>
           <Image
-            src={featuredPack.image}
+            src={`${featuredPack.image}?width=1280&quality=75`}
```

Comment:
So we were using this when we were loading images directly from the CMS which we aren't anymore (in my other PR this was just an `img` tag. We're now using `Image` tag without the cms image loader). What we need here now is actually a `sizes` prop configured to specify widths at various breakpoints.

Here's what this looks like in SPIN now:
https://github.com/rocketinsights/algomart-spin/blob/8d5789779fbfd798026de25da55e04a6568b4a61/apps/web/src/components/featured-pack/featured-pack.tsx#L82

Unfortunately I'm not 100% sure the widths we use on SPIN are exactly the same as what we use here. Luckily, you don't have to specify the widths in pixel terms though so you can instead just resize the window and note whether the image should be approximately `100vw`, `50vw`, `33vw` etc, then use the same media syntax.